tools/ci/npc.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+out=../../npc/scripts_custom.conf
+
+printf "\n" >> $out
```

Comment:
Just to mention it:
You're always appending to the file. If this script is executed more than once, the file would contain duplicates. (But afaik the build bots are completely cleared for every test)